pkg/distro/generic/images.go

```diff
@@ -799,40 +806,16 @@ func iotInstallerImage(workload workload.Workload,
 	// kickstart though kickstart does support setting them
 	img.Kickstart.Timezone, _ = customizations.GetTimezoneSettings()
 
-	instCust, err := customizations.GetInstaller()
-	if err != nil {
-		return nil, err
-	}
-	if instCust != nil && instCust.Modules != nil {
-		img.EnabledAnacondaModules = append(img.EnabledAnacondaModules, instCust.Modules.Enable...)
-		img.DisabledAnacondaModules = append(img.DisabledAnacondaModules, instCust.Modules.Disable...)
-	}
+	img.InstallerCustomizations, err = installerCustomizations(t, bp.Customizations)
 
-	installerConfig, err := t.getDefaultInstallerConfig()
 	if err != nil {
 		return nil, err
 	}
 
-	if installerConfig != nil {
-		img.AdditionalDracutModules = append(img.AdditionalDracutModules, installerConfig.AdditionalDracutModules...)
-		img.AdditionalDrivers = append(img.AdditionalDrivers, installerConfig.AdditionalDrivers...)
-		img.EnabledAnacondaModules = append(img.EnabledAnacondaModules, installerConfig.EnabledAnacondaModules...)
-
-		if menu := installerConfig.DefaultMenu; menu != nil {
-			img.DefaultMenu = *menu
-		}
-
-		if isoroot := installerConfig.ISORootfsType; isoroot != nil {
-			img.RootfsType = *isoroot
-		}
-
-		if isoboot := installerConfig.ISOBootType; isoboot != nil {
-			img.ISOBoot = *isoboot
-		}
-	}
+	// XXX left over bits
```

Comment:
Same here.

Reply:
Left over bits are things that I feel could move into the `installerCustomization` and `InstallerCustomizations` directly, maybe in the future (mostly `img.Kickstart` things). I'll adjust the comment.